examples/withsequence-nested-result.yaml

```diff
@@ -0,0 +1,43 @@
+# This example shows how to use recursion in a Workflow.
```

Comment:
```suggestion
# This example shows how to nested withSequence loops in a Workflow.
```

Since this isn't recursion, the description should be rewritten as well.

Reply:
I had a typo here, sorry. Follow-up comment: https://github.com/argoproj/argo-workflows/pull/12577#discussion_r1498172887